src/taglibs/migrate/common-tag-migrator.js

```diff
@@ -1,8 +1,12 @@
 const fs = require("fs");
 const path = require("path");
-const commonMigrators = fs
-    .readdirSync(path.join(__dirname, "./common/"))
-    .map(dir => require(`./common/${dir}`));
+const getMigrators = d =>
```

Comment:
I think we should revert this such that the `common` migrations is a flat structure (move all the `w-` stuff up a level).

As it stands right now all of these are in an `event-handlers` folder which is not totally accurate and I think unnecessary.

Reply:
Cool. Let me structure it.